aca-docker/mocks/wiremock/__files/prd_users_template_Org-Two.json

```diff
@@ -0,0 +1,19 @@
+{
+  "users": [
+    {
+      "userIdentifier": "",
+      "firstName": "Richard",
+      "lastName": "Solicington",
+      "email": "befta.master.solicitor.richard@gmail.com",
+      "idamStatus": "ACTIVE"
+    },
+    {
+      "userIdentifier": "",
+      "firstName": "Tony",
+      "lastName": "Solicington",
+      "email": "befta.master.solicitor.tony@gmail.com",
+      "idamStatus": "ACTIVE"
+    }
+  ],
+  "organisationIdentifier": "Org-Two"
```

Comment:
NB: all Org-Two references will need to be updated once the AAT group is created. Although we may as well keep the file names so they match confluence.

Reply:
Agreed - I have used `Org-Two` as this was the org id used in feature tests - we can rename accordingly when orgs are created/assigned